### ISSUE TITLE
Add a refreshing background to the app

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#31]
Need a background which changes over time, ofcourse the background should be pokemon related :rofl: 

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by anshumanv** [comment]
Fixed in #43
